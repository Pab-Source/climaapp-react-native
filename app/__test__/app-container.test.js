import React from "react";
import renderer from "react-test-renderer";
import AppContainer from "../app-container";

describe("<AppContainer />", () => {
  const renderApp = renderer.create(<AppContainer />).toJSON();
  test("has 3 child", () => {
    expect(renderApp.children.length).toBe(3);
  });
  test("renders correctly", () => {
    const tree = renderer.create(<AppContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
