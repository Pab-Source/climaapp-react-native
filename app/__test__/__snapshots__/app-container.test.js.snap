// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppContainer /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#488dd8",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexBasis": "20%",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "borderBottomColor": "#ccc",
          "borderBottomWidth": 3,
          "borderStyle": "solid",
          "color": "white",
          "fontSize": 45,
          "marginTop": 40,
        }
      }
    >
      Weather|
      <Text
        style={
          Object {
            "fontSize": 30,
          }
        }
      >
        App
      </Text>
    </Text>
    <View
      style={
        Object {
          "overflow": "hidden",
        }
      }
    >
      <View
        theme={
          Object {
            "colors": Object {
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "primary": "#007aff",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#8F0CE8",
              "success": "#52c41a",
              "warning": "#faad14",
            },
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "transparent",
              "justifyContent": "center",
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flexBasis": "10%",
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "color": "white",
            "marginHorizontal": 15,
            "width": 730,
          }
        }
      >
        <RNCPicker
          items={
            Array [
              Object {
                "label": "Seleccione una ciudad",
                "textColor": 4294880829,
                "value": "Seleccione una ciudad",
              },
              Object {
                "label": "Buenos Aires",
                "textColor": 4294880829,
                "value": "Buenos Aires",
              },
              Object {
                "label": "Londres",
                "textColor": 4294880829,
                "value": "Londres",
              },
              Object {
                "label": "Bogotá",
                "textColor": 4294880829,
                "value": "Bogotá",
              },
              Object {
                "label": "Sao Paulo",
                "textColor": 4294880829,
                "value": "Sao Paulo",
              },
              Object {
                "label": "Lima",
                "textColor": 4294880829,
                "value": "Lima",
              },
            ]
          }
          onChange={[Function]}
          selectedIndex={0}
          style={
            Array [
              Object {
                "height": 216,
              },
              Object {
                "borderBottomWidth": 1,
              },
            ]
          }
        />
      </View>
      <View
        style={
          Object {
            "borderRadius": 3,
            "height": 50,
            "marginHorizontal": 10,
            "overflow": "hidden",
            "width": 730,
          }
        }
      >
        <View
          accessibilityRole="button"
          accessibilityState={
            Object {
              "busy": false,
              "disabled": false,
            }
          }
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#feae3d",
                "borderColor": "#2089dc",
                "borderRadius": 3,
                "borderWidth": 0,
                "flexDirection": "row",
                "justifyContent": "center",
                "padding": 8,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 18,
                  "paddingBottom": 1,
                  "paddingTop": 2,
                  "textAlign": "center",
                }
              }
            >
              Buscar
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flexBasis": "70%",
        "marginTop": 30,
      }
    }
  >
    <RCTScrollView
      horizontal={true}
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
